File: app/src/main/java/com/iamagamedev/mybrandnewgame/GameActivity.java
package com.iamagamedev.mybrandnewgame;

import android.app.Activity;
import android.graphics.Point;
import android.media.AudioManager;
import android.os.Bundle;
import android.view.Display;
import android.view.KeyEvent;

public class GameActivity extends Activity {

    private GameView gameView;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        Display display = getWindowManager().getDefaultDisplay();
        Point size = new Point();
        display.getSize(size);
        setVolumeControlStream(AudioManager.STREAM_MUSIC);

        gameView = new GameView(this, size.x, size.y);
        setContentView(gameView);
    }

    @Override
    protected void onResume() {
        super.onResume();
        gameView.resume();
    }

    @Override
    protected void onPause() {
        super.onPause();
        gameView.pause(isFinishing());
    }

    @Override
    public boolean onKeyDown(int keyCode, KeyEvent event) {
        if (keyCode == KeyEvent.KEYCODE_BACK) {
            finish();
            return true;
        }
        return false;
    }
}

File: app/src/main/java/com/iamagamedev/mybrandnewgame/GameActivity.kt
package com.iamagamedev.mybrandnewgame

import android.app.Activity
import android.graphics.Point
import android.media.AudioManager
import android.os.Bundle
import android.view.KeyEvent

class GameActivity : Activity() {
    private var gameView: GameView? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val display = windowManager.defaultDisplay
        val size = Point()
        display.getSize(size)
        volumeControlStream = AudioManager.STREAM_MUSIC
        gameView = GameView(this, size.x, size.y)
        setContentView(gameView)
    }

    override fun onResume() {
        super.onResume()
        gameView!!.resume()
    }

    override fun onPause() {
        super.onPause()
        gameView!!.pause(isFinishing)
    }

    override fun onKeyDown(keyCode: Int, event: KeyEvent): Boolean {
        if (keyCode == KeyEvent.KEYCODE_BACK) {
            finish()
            return true
        }
        return false
    }
}